### DISCUSSION TITLE
ISS is not in Stellarium

### DISCUSSION THREAD

**Discussion by Anthrwpos** [Stellarium/stellarium#524] [Q&A] [answered]
Hi, i have Stel. ver 0.18.2 and i can't find anywhere the ISS. You have deleted it or is something i can do to find it?
Thanks

**Comment by gzotti** [comment, marked as answer]
Did you update the satellite elements?

**Comment by alex-w** [comment]
Did you enabled the visualization of satellites on bottom bar?

**Comment by Anthrwpos** [comment]
@gzotti Thanks, it was so easy after seeing the solution.

**Comment by Anthrwpos** [comment]
@alex-w Yes, from the start, it was the update as gzotti wrote. Thanks for the suggestion.
